src/class-metadata.php

```diff
@@ -517,7 +517,8 @@ private function get_coauthor_names( int $post_id ): array {
 					if ( ! empty( $post_author ) ) {
 						$coauthors[] = $post_author;
 					}
-				} // the empty else case is because if we force guest authors, we don't ever care what value wp_posts.post_author has.
+				}
+				// The empty else case is because if we force guest authors, we don't ever care what value wp_posts.post_author has.
```

Comment:
I think I understand what that comment means, but I find it a bit confusing. If it needs to be there, maybe there's a way to simplify it or make it more clear?

Reply:
I just reformatted the line, not changed the comment itself. I agree it could be improved, but I think this is out of scope for this PR.